internal/httpclient/api/openapi.yaml

```diff
@@ -1841,6 +1841,45 @@ paths:
         sessions tha belongs to a given Identity.
       tags:
       - v0alpha1
+    get:
+      description: |-
+        This endpoint is useful for:
+
+        Issuing session or session token for a given identity without authenticating
```

Comment:
I see that it's a bit of a convention to phrase it like this, but I wouldn't phrase it this way. It's not *useful* for doing that, it's what it does. :P

I guess what doing that is useful for is migrations from other auth system for example.

Reply:
Yeah, that argument makes sense only if we propose this upstream and don't intend on reverting that after the migration.

Reply:
I don't think this is only single-purpose endpoint so I didn't want to phrase it this way, it will be used by us for system migrations but could also be used for eg. impersonations.